src/coreclr/debug/dbgutil/elfreader.cpp

```diff
@@ -127,7 +127,7 @@ ElfReader::PopulateForSymbolLookup(uint64_t baseAddress)
     // Enumerate program headers searching for the PT_DYNAMIC header, etc.
     if (!EnumerateProgramHeaders(
         baseAddress,
-#if defined(TARGET_LINUX_MUSL) || defined(TARGET_RISCV64)
+#if defined(TARGET_LINUX_MUSL) || defined(TARGET_RISCV64) || defined(HOST_ANDROID)
```

Comment:
```suggestion
#if defined(TARGET_LINUX_MUSL) || defined(TARGET_RISCV64) || defined(TARGET_ANDROID)
```
TARGET_ANDROID for consistency with the existing conditions?